PlaceRouteHierFlow/placer/Placer.cpp

```diff
@@ -233,6 +233,17 @@ std::map<double, std::pair<SeqPair, ILP_solver>> Placer::PlacementCoreAspectRati
     logger->debug("Random number generator seed={0}", seed);
   }
 
+  // Perturb as the very first sequence pair may not be legal
```

Comment:
@854768750 , @srini229 , @stevenmburns : No feasible sequence pair is found even after 1e5 perturbations to the initial sequence pair. Even the initial sequence pair is found using different algorithms, the current perturbation scheme seems unlikely to generate a new, legal sequence.